src/manager/InMemoryTaskManager.java

```diff
@@ -14,7 +14,7 @@ public class InMemoryTaskManager implements TaskManager {
     private final Map<Integer, Task> tasks = new HashMap<>();
     private final Map<Integer, Subtask> subtasks = new HashMap<>();
     private final Map<Integer, Epic> epics = new HashMap<>();
-    private InMemoryHistoryManager historyManager = Managers.getDefaultHistory();
+    private final InMemoryHistoryManager historyManager = Managers.getDefaultHistory();
```

Comment:
Нужно обозначать интерфейс, а создавать объект уже с определенной реализацией
```suggestion
    private final HistoryManager historyManager = Managers.getDefaultHistory();
```

Reply:
fixed, thanks